src/veh_interact.cpp

```diff
@@ -572,13 +589,13 @@ void veh_interact::do_install()
     }
     mvwprintz(w_mode, 0, 1, c_ltgray, _("Choose new part to install here:"));
     wrefresh (w_mode);
-    
+
```

Comment:
Yea that's what I meant, that space should have been cleaned when I edited the file. I noticed C::B did it for me in several other places, for some reason it appears to have ignored this one. Apologies for the trouble that caused.